packages/dds/tree/src/simple-tree/toFlexSchema.ts

```diff
@@ -78,6 +84,17 @@ export function toFlexSchema(root: ImplicitFieldSchema): FlexTreeSchema {
 	return typed;
 }
 
+/**
+ * Converts a {@link ImplicitFieldSchema} into a {@link TreeStoredSchema}.
+ */
+export function toStoredSchema(root: ImplicitFieldSchema): TreeStoredSchema {
```

Comment:
This skips over flex tree schema: code using this will need less changes as we remove flex tree schema.